nipype/interfaces/fsl/model.py

```diff
@@ -693,24 +690,23 @@ def _get_numcons(self):
             fp.close()
         return numtcons, numfcons
 
-    def _list_outputs(self):
-        outputs = self._outputs().get()
+    def _post_run(self):
         cwd = os.getcwd()
         results_dir = os.path.join(cwd, self.inputs.results_dir)
-        outputs['results_dir'] = results_dir
+        self.outputs.results_dir = results_dir
         pe_files = self._get_pe_files(results_dir)
         if pe_files:
-            outputs['param_estimates'] = pe_files
-        outputs['residual4d'] = self._gen_fname('res4d.nii', cwd=results_dir)
-        outputs['dof_file'] = os.path.join(results_dir, 'dof')
-        outputs['sigmasquareds'] = self._gen_fname('sigmasquareds.nii',
+            self.outputs.param_estimates = pe_files
+        self.outputs.residual4d = self._gen_fname('res4d.nii', cwd=results_dir)
+        self.outputs.dof_file = os.path.join(results_dir, 'dof')
+        self.outputs.sigmasquareds = self._gen_fname('sigmasquareds.nii',
                                                    cwd=results_dir)
-        outputs['thresholdac'] = self._gen_fname('threshac1.nii',
+        self.outputs.thresholdac = self._gen_fname('threshac1.nii',
                                                  cwd=results_dir)
         if Info.version() and LooseVersion(Info.version()) < LooseVersion('5.0.7'):
-            outputs['corrections'] = self._gen_fname('corrections.nii',
+            self.outputs.corrections = self._gen_fname('corrections.nii',
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unexpected keyword argument 'cwd' in method call](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/2065944001/issues/source?bid=2767207&fileBranchId=2890884#l707)